lib/workers/global/config/parse/env.ts

```diff
@@ -164,5 +165,5 @@ export function getConfig(inputEnv: NodeJS.ProcessEnv): AllConfig {
 
   unsupportedEnv.forEach((val) => delete env[val]);
 
-  return config;
+  return massageConfig(config);
```

Comment:
this should be a seperate PR

Reply:
This has been part or the original PR, but I going to create a new PR for this.